src/Html.php

```diff
@@ -1292,6 +1292,14 @@ public static function includeHeader(
             if (in_array('dashboard', $jslibs)) {
                 $tpl_vars['css_files'][] = ['path' => 'css/standalone/dashboard.scss'];
                 Html::requireJs('dashboard');
+
+                // include more js libs
+                $jslibs = array_merge($jslibs, [
+                    'gridstack',
+                    'charts',
+                    'clipboard',
+                    'sortable'
+                ]);
```

Comment:
It works because added libraries handling is made after this piece of code, but if order of these `if (in_array(XXX, $js_libs))` change in the future, it cause issues.
To prevent such issues, following piece of code should be moved on line 1267:
```
            if (in_array('dashboard', $jslibs)) {
                // include more js libs
                $jslibs = array_merge($jslibs, [
                    'gridstack',
                    'charts',
                    'clipboard',
                    'sortable'
                ]);
            }
```